ocean_lib/web3_internal/wallet.py

```diff
@@ -32,18 +33,21 @@ class Wallet:
     node since we only send the raw transaction hash so the user info is safe.
 
     Usage:
-        1. `wallet = Wallet(ocean.web3, private_key=private_key)`
+        1. `wallet = Wallet(ocean.web3, private_key=private_key, block_confirmations=ocean.config.block_confirmations)`
 
     """
 
     _last_tx_count = dict()
 
     @enforce_types
-    def __init__(self, web3: Web3, private_key: str) -> None:
+    def __init__(
+        self, web3: Web3, private_key: str, block_confirmations: Integer
+    ) -> None:
```

Comment:
Make this such that it can handle just a plain int, or the new Integer object.

Reply:
But if plain int comes in, convert to Integer objects before storing.

Reply:
I love this idea. It'll be in my next push.

Reply:
+1